test/functorch/test_control_flow.py

```diff
@@ -3545,6 +3545,144 @@ def cond_fn(x):
             self.assertEqual(cnt.frame_count, 3)
 
 
+_hop_schema_test_dtypes = [
```

Comment:
nit: this isn't a "dtype", call it something else

Reply:
Also I would match the casing in the schema anyways (SymInt instead of symint) to make it more consistent